app/src/main/java/com/example/simplenavigation/MainActivity.java

```diff
@@ -25,28 +25,56 @@
     private Robot mRobot;
     private final String TAG = MainActivity.class.getSimpleName();
 
+    /**
+     * Function getInt is used to get integer value from EditText views
+     * @param viewId is used to find view we want to get value from
+     * @return is value of view or if view is empty function returns "50" so robot ends up facing upwards towards user
+     */
+    public int getInt(int viewId) {
+        EditText text = findViewById(viewId);
+        String tiltVal = text.getText().toString();
+        if (tiltVal.equals("")) {
```

Comment:
I suggest using `.isEmpty()` instead of `.equals("")`. It's more clear and captures some edge-cases.

Reply:
Used 
`tiltVal.isEmpty()`

And same for function getFload
`floatVal.isEmpty()`